pkg/engine/destroy.go

```diff
@@ -80,8 +80,8 @@ func newDestroySource(
 
 	// Like Update, if we're missing plugins, attempt to download the missing plugins.
 
-	if err := ensurePluginsAreInstalled(ctx, opts, plugctx.Diag, plugins.Deduplicate(),
-		plugctx.Host.GetProjectPlugins()); err != nil {
+	if err := EnsurePluginsAreInstalled(ctx, opts, plugctx.Diag, plugins.Deduplicate(),
+		plugctx.Host.GetProjectPlugins(), false, false); err != nil {
```

Comment:
```suggestion
		plugctx.Host.GetProjectPlugins(), false /*reinstall*/, false /*explicitInstall*/); err != nil {
```

Also in the other places where we pass these as naked true/false